src/test/java/com/google/javascript/cl2dts/types.js

```diff
@@ -59,3 +59,13 @@ types.j = {a: 'a'};
  * @template K,V
  */
 types.objectWithGenericKeyType = function(obj) { };
+
+/**
+ * @const
```

Comment:
Without const, the symbol doesn't appear in the topScope (I assume because it gets removed because it's unused). @export would work just as well